### DISCUSSION TITLE
Bonus Claim Issue

### DISCUSSION THREAD

**Discussion by AbdulMoeezBaig** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#593] [General]
Recently the script stopped collecting the 5 minute points bonus , now it collects only the 3-4 minutes (+10) and not the +50.
Anyone else having the issue?

**Comment by Rakambda** [comment]
#592

**Comment by AbdulMoeezBaig** [comment]
Can someone mention how to implement the fix? [is it just replacing the 2 mentioned files or is there more to it?]
